server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/PlanFragment.java

```diff
@@ -94,12 +96,14 @@ private PlanNode getPlanNodeById(PlanNode root, PlanNodeId nodeId) {
     return null;
   }
 
-  public static PlanFragment deserialize(ByteBuffer byteBuffer) throws IllegalPathException {
+  public static PlanFragment deserialize(ByteBuffer byteBuffer)
+      throws IllegalPathException, AuthException {
```

Comment:
why does add AuthException  in deserialize mothod?

Reply:
I have made changes to remove the AuthorException